ROADMAP.md

```diff
@@ -2,92 +2,113 @@ Roadmap
-   meaningful security guarantees. Batteries included.
-
-## Tighter spec compliance
-
-The in-toto specification drives the behavior of all of our implementations,
-yet 100% compliance is not yet achieved by all of them. This year we will focus
-efforts on making sure that the existing implementations follow the
-specification to the word.
-
-## Maturing of existing sub-projects
-
-The in-toto project has sprouted to host multiple implementations, plugins, cli
-tools, helper commands, ansible playbooks, etc. This year we intend to mature
-at least the following components:
-
-### Jenkins plugin
+### Kubernetes components
 
-The official Jenkins plugin is already usable, yet some features like
-stdin/stdout registration are desirable, other environment information is
-worth having.
+### in-toto Jenkins plugin
 
-This also calls for a more mature Java implementation, with higher test
-coverage and a more user-friendly API.
+The [in-toto Jenkins plugin](https://github.com/jenkinsci/in-toto-plugin) is
+used to generate in-toto links for steps in Jenkins pipelines. During the last
+roadmap period, we added Grafeas as a native transport. During this next
+roadmap, we aim to work on further maturing the plugin with greater granularity,
+particularly in better specifying the artifacts to record while generating the
+attestations, and some other refactoring as recorded in the repository's issues.
 
-### Kubernetes components
+## TUF + in-toto
 
-The kubernetes admission controller will greatly benefit from a resource-type
-identifier, and proper handling of container images. Given it's cloud-native
-nature, verification will mostly run over containers that are about to be
-admitted into an organization's cloud.
+We are continuing to work on better integrating in-toto with our sister project,
+The Update Framework (TUF).
 
-This will also call for a smarter interaction between our go implementation and
-containerd/runc to intelligently handle container resource-types.
+### ITE-2 and ITE-3
 
-### GPG support
+Trishank Karthik Kuppusamy (@trishankatdatadog) sponsored
+[ITE-2](https://github.com/in-toto/ITE/blob/master/ITE/2/README.adoc) and
+[ITE-3](https://github.com/in-toto/ITE/blob/master/ITE/3/README.adoc) that
+detail how TUF and in-toto can be combined to provide end-to-end security
+guarantees. We aim to work with the stakeholders of these ITEs to standardize 
+the recommendations.
 
-We intend to move the GPG components to securesystemslib and standardize this as
-either a TAP or an ITE so our sister project, TUF can benefit from GPG support.
+### Streamline development of overlapping components
 
-### Apt transport and rebuilders
+During this next year, we plan to propose an ITE (and perhaps a TAP) to
+coordinate the development of the overlapping parts of the specification. The
+two projects share parts of the document formats, and it is beneficial to
+maintain consistency.
```

Comment:
I'd be interested in collaborating on this!

Reply:
That'd be fantastic!